src/Services/UserService.jsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable guard-for-in */
```

Comment:
Remove 

/* eslint-disable no-restricted-syntax */
/* eslint-disable no-else-return */
/* eslint-disable prefer-promise-reject-errors */
/* eslint-disable no-unused-vars */

Reply:
or explain why it's mandatory

Reply:
change users pour être un tableau plutôt qu'un objet et fait un for of. L'id n'est pas nécessaire en clef, il est dans l'objet

Reply:
et il est possible d'utiliser un .find pour que ce soit plus efficace